core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.spec.ts

```diff
@@ -28,11 +28,17 @@ import { SiteAnalyticsService } from 'services/site-analytics.service';
 import { ThreadDataBackendApiService } from 'pages/exploration-editor-page/feedback-tab/services/thread-data-backend-api.service';
 import { UserService } from 'services/user.service';
 import { UserInfo } from 'domain/user/user-info.model';
+import { RteOutputDisplayComponent } from 'rich_text_components/rte-output-display.component';
 
 class MockChangeDetectorRef {
   detectChanges(): void {}
 }
 
+enum ExpansionTabType {
+  CONTENT,
+  TRANSLATION
+}
```

Comment:
Define this only in one place and not in two.

Reply:
Done